src/domains/idpay-app/api/idpay_appio_full/openapi.appio.full.yml

```diff
@@ -1618,6 +1618,24 @@ paths:
               example:
                 code: PAYMENT_NOT_FOUND_OR_EXPIRED
                 message: transaction not found or expired
+        '412':
+          description: The transaction version mismatch
```

Comment:
I think is more appropriate to return a REJECTED transaction when the citizen counters have a etag mismatch, cause the reward could be not guaranteed

Reply:
@petretiandrea  thus we should return 200 with REJECTED as transaction status and the PAYMENT_TRANSACTION_VERSION_MISMATCH as rejection reason?

Reply:
Something like PAYMENT_CANNOT_GUARANTEE_REWARD?

Reply:
The actual implementation, in case of version mismatch, will calculate again the reward and return this error if the reward is certainly different.
the error code just used "PAYMENT_TRANSACTION_VERSION_MISMATCH" is a hint to suggest the wrong etag provided.
Thus do you want to rename this code into "PAYMENT_CANNOT_GUARANTEE_REWARD"?

Reply:
Yes, but in both cases the error is due to the fact that the reward cannot be guaranteed. Even in the case of a recalculation if the reward is different it is not provided because it is not guaranteed compared to what is shown in the preview, don't you agree?